generators/server/templates/build.gradle.ejs

```diff
@@ -387,9 +387,8 @@ dependencies {
     implementation("io.micronaut.email:micronaut-email-template")
     implementation("io.micronaut.views:micronaut-views-thymeleaf")
 
-    implementation "org.zalando:problem:$problem_version"
     implementation "org.zalando:problem-violations:$problem_version"
-    implementation "org.zalando:jackson-datatype-problem:$problem_version"
+    implementation "org.zalando:jackson-datatype-problem:0.27.1"
```

Comment:
Shouldn't it be the same version as in maven, so `0.29.1`?

Reply:
Latest is 0.27.1.
See https://mvnrepository.com/artifact/org.zalando/jackson-datatype-problem
We should extract versions to dependabot later.

Reply:
Ah sorry, mixed it up with `org.zalando:problem-violations`